index.js

```diff
@@ -0,0 +1,565 @@
+    this.emitWithState("Start");
+  },
+  "AMAZON.CancelIntent": function () {
+    this.response.speak('Auf Wiederhören!');
+    this.emit(":responseReady");
+  },
+  "AMAZON.HelpIntent": function () {
+    this.response.speak('Auf Wiederhören!');
+    this.emit(":responseReady");
+  },
+  "AMAZON.YesIntent": function () {
+    this.handler.state = "_START";
+    this.emitWithState("Start");
+  },
+  "AMAZON.NoIntent": function () {
+    this.response.speak('Auf Wiederhören!');
+    this.emit(":responseReady");
+  },
+  "HomeIntent": function () {
+    this.handler.state = "_START";
+    this.emitWithState("Start");
+  },
+  "Unhandled": function () {
+    this.emitWithState("Music");
+  }
+});
+
+const lection_handlers = Alexa.CreateStateHandler("_LECTION", {
+  'Lection': function () {
+    db.getChapterFromUser( this.event.session.user.userId, (chapter) => {
+      this.attributes["unhandled"] = false;
+      if(chapter == 1){
+        this.response.speak("Du bist am Anfang, sage fortfahren um zu starten").listen("Möchtest du das Kapitel fortfahren oder ein bestimmtes Kapitel hören?");
+      } else {
+        this.response.speak("Du hast Kapitel: "+parseInt(chapter-1)+" gehört, Möchtest du das Kapitel fortfahren oder ein bestimmtes Kapitel hören?").listen("Möchtest du das Kapitel fortfahren oder ein bestimmtes Kapitel hören?");
+      }
+      this.emit(":responseReady");
+    });
+  },
+  'LectureNumberIntent': function () {
+    if(this.attributes["unhandled"]){
+      this.attributes["unhandled"] = false;
+      this.attributes["continuelecture"] = false;
+
+      //console.log(hf.getNumberToStringValue(this.event.request.intent.slots.Answer.value.toLowerCase()));
+
+      if(this.event.request.intent.name == "GameContinueIntent"){
+        this.emitWithState("LectureContinueIntent");
+      } if(this.event.request.intent.name == "GameIntent"){
+        this.emitWithState("LecturePlayIntent");
+      } else {
+        this.response.speak("Das hat nicht geklappt, aktuell sind 2 Kapitel verfügbar.").listen("Sage zum Beispiel Kapitel Zwei oder Kapitel fortfahren.");
+        this.emit(":responseReady");
+      }
+
+    } else {
+      this.attributes["unhandled"] = false;
+      this.attributes["continuelecture"] = false;
+      this.response.speak("Mit welchem Kapitel möchtest du starten?").listen("Sage zum Beispiel Kapitel Zwei.");
+      this.emit(":responseReady");
+    }
+  },
+  'LecturePlayIntent': function () {
+
+    console.log("CONTINUE: " + this.attributes["continuelecture"] + "\n");
+
+    db.getChapterFromUser( this.event.session.user.userId, (chapter) => {
+
+      if(this.attributes["continuelecture"]){
+        if(chapter >1){
+          this.attributes["continuelecture"] = false;
+
+          if(chapter==3){
+            db.updateChapter(this.event.session.user.userId, 1);
+            chapter = 1;
+          } else {
+            db.updateChapter(this.event.session.user.userId, (parseInt(chapter)+1));
```

Comment:
`chapter` is threaded as `int` before. Why parse it only here?

Reply:
I should remove that parse attempt